app/models/answer.rb

```diff
@@ -1,6 +1,12 @@
 class Answer < ActiveRecord::Base
   belongs_to :question
 
+  def add_answer answer
+    json = answer.to_json
+    response = RestClient.post(ENV["qm_api_url"] + "questions/#{params[:question_id]}/answers", json, :content_type => :json , :accept => :json)
+    logger.debug "THIS IS THE #{response}"
```

Comment:
While we're updating this, let's make these `logger.debug` statements more useful maybe?